build-scripts/multidirectory.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+# Creates/removes/updates branches according the `jobs/` directory on the
+# master branch.  Local files and branches will be modified, so this script
+# should be executed on a clean, dedicated clone.
+set -eu
+
+set_difference() {
+    printf '%s\n%s\n%s\n' "$1" "$1" "$2" | sort | uniq -u
+}
+
+list_branches() {
+    git branch --list --remotes 'origin/*' \
+        | awk '$1 ~ /^origin\/.*\// {print gensub("^origin/", "", 1, $1)}'
+}
+
+list_jobs() {
+    [ -e jobs/ ] || return 0
+    (cd jobs/ && find * -name Jenkinsfile | sed 's,/Jenkinsfile$,,')
+}
+
+filter_unchanged() {
```

Comment:
This doesn't seem super useful -- use `git` to your advantage here, always do the copy/clobber and check if there is anything to commit.